setup.cfg

```diff
@@ -1,7 +1,57 @@
+[flake8]
```

Comment:
_FYI:_ I have fixed the order of configs (alphabetical) - thanks to that, it will be easier to keep that up to date with other repositories 😉